spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -439,8 +463,12 @@ def step(  # noqa
         print(f"base_action: {arr2str(base_action)}\tarm_action: {arr2str(arm_action)}")
 
         # Spin until enough time has passed during this step
+        # TODO: semantic place ee: temp hack to command this out
+        # for not using base velocity
         start_time = time.time()
-        if base_action is not None or arm_action is not None:
+        if arm_ee_action is not None:
+            print("do not move the base")
```

Comment:
remove print

Reply:
do we also need to change this to:
`if arm_ee_action is not None and self._max_lin_dist_scale == 0.0`?
I thought we needed both conditions to toggle if the base moves or not